Backend/all/login

nakagame opened the pull request:
[Backend/All/Login] -  implementing manual multiple login functionality
- Created 2 Controllers for multiple login - Auth/SellerLoginController and Auth/AdminLoginController   
- Create 2 Controllers for showing blade - Users/SellerController and Users/AdminController
- Modified web.php - grouping route

nakagame committed:
[Backend/All/Login] -  implementing manual multiple login functionality
- Created 2 Controllers for multiple login - Auth/SellerLoginController and Auth/AdminLoginController
- Create 2 Controllers for showing blade - Users/SellerController and Users/AdminController
- Modified web.php - grouping route

nakagame committed:
Merge branch 'main' into Backend/All/Login

nakagame committed:
[Backend/All/Login] -  implementing manual multiple login functionality
- Created 2 Controllers for multiple login - Auth/SellerLoginController and Auth/AdminLoginController
- Create 2 Controllers for showing blade - Users/SellerController and Users/AdminController
- Modified web.php - grouping route

nakagame committed:
Merge branch 'main' into Backend/All/Login

Ninit29 commented on app/Http/Controllers/Users/SellerController.php:
```diff
@@ -0,0 +1,22 @@
+{
+    private $seller;
+
+    public function __construct(Seller $seller)
```
![image](https://github.com/kredo-school/18tha-gj_mall/assets/62535075/9883c830-8c13-4404-bd2e-082203ab3c93)

This part of the code is not being used. You may want to remove it and just add it when you use it.

Ninit29 commented on app/Http/Controllers/Users/AdminController.php:
```diff
@@ -0,0 +1,22 @@
+    private $admin;
+
+    public function __construct(Admin $admin)
+    {
```
![image](https://github.com/kredo-school/18tha-gj_mall/assets/62535075/8bfcec63-3524-4f6e-bf01-fa8ea87f88e7)

This part of the code is not being used. You may want to remove it and just add it when you use it.

nakagame committed:
[Backend/All/Login] - Delete unnecessary code
- AdminLoginController - delete __construct
- SellerLoginController - delete __construct

nakagame committed:
Merge branch 'main' into Backend/All/Login

nakagame committed:
Merge branch 'Backend/All/Login' of github.com:kredo-school/18tha-gj_mall into Backend/All/Login
